lib/output/invalid.go

```diff
@@ -0,0 +1,13 @@
+package output
+
+import (
+	"fmt"
+)
+
+type InvalidOutput struct {
+	Config Config
+}
+
+func (o *InvalidOutput) GenerateOutput(elements interface{}) (string, error) {
```

Comment:
So, here you're ['struct typing'](https://medium.com/@matryer/golang-advent-calendar-day-one-duck-typing-a513aaed544d), rather than explicitly declaring an `type Outputer interface`, right?